bundle/android/strings.xml

```diff
@@ -309,9 +313,6 @@ translators: %s: Select control button label e.g. "Button width" -->
     <string name="gutenberg_native_replace_video" tools:ignore="UnusedResources">Replace video</string>
     <string name="gutenberg_native_retry" tools:ignore="UnusedResources">Retry</string>
     <string name="gutenberg_native_rich_text_editing" tools:ignore="UnusedResources">Rich text editing</string>
-    <!-- translators: %s: Block name e.g. "Image block"
-translators: Block name. %s: The localized block name -->
-    <string name="gutenberg_native_s_block" tools:ignore="UnusedResources">%s block</string>
```

Comment:
No longer needed as it's now used in the web version ([reference](https://github.com/WordPress/gutenberg/blob/980cad0d37b827dab245797c8d67cf90d0505256/packages/block-editor/src/components/global-styles/get-global-styles-changes.js#L56)).